trinity/_utils/logging.py

```diff
@@ -105,6 +107,17 @@ def setup_trinity_file_and_queue_logging(
 
     log_queue = ctx.Queue()
 
+    formatted_logfile = str(logfile_path)
+    str_timestamp = datetime.now().strftime('_%Y%m%d_%H%M%S')
+    idx = formatted_logfile.rfind(
+        ".",
+        len(formatted_logfile) - len(os.path.basename(formatted_logfile))
+    )
+    if idx != -1:
+        formatted_logfile = formatted_logfile[0:idx] + str_timestamp + formatted_logfile[idx:]
+    else:
+        formatted_logfile = formatted_logfile + str_timestamp + ".log"
+
     handler_file = RotatingFileHandler(
         str(logfile_path),
```

Comment:
Could it be better to do it with `Path.suffix`? For example something like this:
```suggestion
         str(logfile_path)[:-len(logfile_path.suffix)] + str_timestamp + logfile_path.suffix,
```